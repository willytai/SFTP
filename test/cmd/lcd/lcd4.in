lcd lls/dirforcheck
lls
lcd ../../
