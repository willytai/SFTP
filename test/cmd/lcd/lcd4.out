sftp> lcd lls/dirforcheck
sftp> lls
file2.txt file3.txt file1.txt 
sftp> lcd ../../
sftp> 
